.github/workflows/lint.yaml

```diff
@@ -24,6 +24,17 @@ jobs:
         run: pip install -r ${GITHUB_WORKSPACE}/requirements.txt
       - name: Download Resources
         run: python ${GITHUB_WORKSPACE}/download_resources.py
+      - name: Download GN via CIPD
```

Comment:
Consider doing this step before the checkout step as it doesn't depend on anything else

Reply:
Done